projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -281,6 +281,15 @@ public function add_gutenberg_menus( $wp_admin = false ) {
 		parent::add_gutenberg_menus( $wp_admin );
 	}
 
+	/**
+	 * Always use WP Admin for comments.
+	 *
+	 * @param bool $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
+	 */
+	public function add_comments_menu( $wp_admin = false ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
Is the param `$wp_admin` necessary if we are always passing in `true` below?

Reply:
Copying @BogdanUngureanu excellent explanation

> yup, it’s required because PHP 7.4 and below will throw a warning if you change the signature of a method you override in a child. Also, as of PHP 8 this will throw a fatal error.